src/ComputeSharp/Graphics/Extensions/Interop/ID3D12DeviceExtensions.cs

```diff
@@ -362,6 +363,11 @@ public static ComPtr<ID3D12CommandQueue> CreateCommandQueue(this ref ID3D12Devic
         d3D12CommandQueueDesc.Flags = D3D12_COMMAND_QUEUE_FLAG_NONE;
         d3D12CommandQueueDesc.NodeMask = 0;
 
+        if (Configuration.IsGpuTimeoutDisabled)
+        {
+            d3D12CommandQueueDesc.Flags = D3D12_COMMAND_QUEUE_FLAG_DISABLE_GPU_TIMEOUT;
```

Comment:
might want to use `|=` ? Not technically necessary right now, but if you add more flags here you might get stuck in a debugging black hole until you notice that this is `=` instead of `|=` :)